https://launchpadlibrarian.net/204464340/0002-fix-webkit2-build-breakage.patch

webkit_web_view_get_main_frame() is a webkit1 API function.
Using it without a check breaks webkit2 builds.

Signed-off-by: Gustavo Zacarias <gustavo@zacarias.com.ar>

diff -Nura midori-0.5.10.orig/midori/midori-view.c midori-0.5.10.wk2/midori/midori-view.c
--- midori-0.5.10.orig/midori/midori-view.c	2015-04-24 08:50:24.940501790 -0300
+++ midori-0.5.10.wk2/midori/midori-view.c	2015-04-25 07:19:21.707132362 -0300
@@ -1256,9 +1256,11 @@
                                         message, error->message, NULL,
                                         _("Try Again"), web_frame);
 
+#ifdef WEBKIT2
     /* if the main frame for the whole tab has a network error, set tab error status */
     if (web_frame == webkit_web_view_get_main_frame (web_view))
         midori_tab_set_load_error (MIDORI_TAB (view), MIDORI_LOAD_ERROR_NETWORK);
+#endif
 
     g_free (message);
     g_free (title);
